hubblestack/files/hubblestack_nova/stat_nova.py

```diff
@@ -177,3 +209,80 @@ def _get_tags(data):
                     formatted_data.pop('data')
                     ret[tag].append(formatted_data)
     return ret
+
+def _check_mode(  max_permission , given_permission, allow_more_strict):
+    '''
+    This function checks whether a file's permission are equal to a specified set of permission or more restrictive. The function returns
```

Comment:
Lets reduce the doc staring here and break into multiple lines rather than big lines.